subsys/net/lib/zperf/zperf_udp_uploader.c

```diff
@@ -309,6 +310,16 @@ int zperf_udp_upload(const struct zperf_upload_params *param,
 		return sock;
 	}
 
+	if (param->if_name[0]) {
+		(void)memset(req.ifr_name, 0, sizeof(req.ifr_name));
+		strcpy(req.ifr_name, param->if_name);
```

Comment:
Better use `strncpy()` here like you did in other similar calls above. This certainly works, but there are scanning tools that will start to complain for these string copy functions that it thinks are unsafe.

Reply:
updated